reference/orders.v3.yml

```diff
@@ -2280,9 +2280,15 @@ definitions:
   Amount:
     type: number
     format: float
-    description: A non-negative 2 decimal place rounded value that represents that amount that can be charged/refunded via payment providers
+    description: A non-negative 2 decimal place rounded value that represents the amount that can be charged/refunded via payment providers.
     example: 1.99
     title: Amount
+  AdjustmentAmount:
+    type: number
+    format: float
+    description: A negative or positive 2 decimal place rounded value that represents the difference between the refund amount requested in the refund quote and the actual amount that is refundable on the order. This value is negative when the refund amount requested in the refund quote is more than the total refundable amount. This value is positive when the total refundable amount has increased, e.g. as a result of rounding.
+    example: -10.20
+    title: Amount
```

Comment:
should title be `adjustment amount` ?

Reply:
Updated in latest [commit](https://github.com/bigcommerce/api-specs/pull/293/commits/ffa781f12ede6bb409ef7ffa92faf45535a27674).